### PR TITLE
devex: vscode: Update extension to OE v0.11.0

### LINKED ISSUE #3630: Update VS Code extension to use latest OE SDK
The VS Code extension just uses whatever OE SDK version is already installed so is already agile in that sense, but the project templates are old (e.g., they don't import platform.edl or any more specific EDL) and it apparently references the feature.new_platforms branch rather than the master branch.

Given that the IoT Edge enclave support just announced GA and the [announcement ](https://techcommunity.microsoft.com/t5/internet-of-things/secure-your-iot-edge-compute-today-with-enclaves/ba-p/1685143) points to the VS Code extension, this should be cleaned up.

### PR DESCRIPTION
**Pull request by HernanGatta**
## Overview

Update the Visual Studio Code extension with support for Open Enclave v0.11.0 both for SGX as well as for OP-TEE on ARM TrustZone for QEMU AArch64/ARMv8 and the Scalys Grapeboard (LS1012A).
 
### Extension changes

Building the extension's VSIX now relies on the proper execution of WebPack instead of carrying all of its Node module dependencies separately. In addition, the version of the DevKit was bumped to ensure that the old DevKit, while remaining present on the system in case the user has projects that use it, is upgraded to the new one targeting the new version of the SDK.

_Extension version bump to 2.1.0._

### SGX changes

The extension now creates a new build output directory for SGX for enclaves with LVI mitigations. Additionally, new build and launch tasks are provided to compile and execute the resulting host and enclave pair. Instructions are added to the README for the user to install the LVI-aware toolchain.

### OP-TEE changes

The DevKit tarball carries builds of the SDK at v0.11.0. No further changes were made.

Resolves #3630.

Signed-off-by: Hernan Gatta <hegatta@microsoft.com>

### CODE REVIEW

File: devex/vscode-extension/src/common/constants.ts [hunk 1]
```diff
@@ -10,6 +10,11 @@ export class Constants {
     //
     public static ExtensionId = "ms-iot.msiot-vscode-openenclave";
 
+    //
+    // SDK version support
+    //
+    public static SdkVersion = "v0.11.0";
```
Comment (bfjelds) [inline devex/vscode-extension/src/common/constants.ts:16]: why introduce another version?  what does DevKitVersion mean with SdkVersion?

File: devex/vscode-extension/.vscodeignore [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+**/tslint.json
+**/webpack.config.json
 3rdparty/**
 devkit/**
+node_modules/**
+out/**
 scripts/**
+src/**
+vsc-extension-quickstart.md
+webpack.config.json
```
Comment (dthaler) [inline devex/vscode-extension/.vscodeignore:17]: nit: add newline

File: devex/vscode-extension/package.json [hunk 3]
```diff
@@ -96,4 +101,4 @@
 		"webfreak.debug",
 		"microsoftdebuggingplatform.vscode-cdb"
 	]
-}
+}
```
Comment (dthaler) [inline devex/vscode-extension/package.json:104]: nit: restore newline

File: devex/vscode-extension/src/common/constants.ts (hunk 1 above)
Comment (dthaler) [inline devex/vscode-extension/src/common/constants.ts:16]:
```suggestion
    public static OESdkVersion = "v0.11.0";
```

**Review by dthaler — APPROVED** [review]

**Comment by radhikaj** [comment]
Bors try

**Comment by radhikaj** [comment]
Bors try

File: devex/vscode-extension/src/common/constants.ts (hunk 1 above)
Comment (HernanGatta) [reply devex/vscode-extension/src/common/constants.ts:16]: Oh, interesting. Would it then make sense to merely bump `DevKitVersion` to `3.0.0.0`? It seems to me that I reinvented a solution you had already implemented for the same problem.

Comment (HernanGatta) [reply devex/vscode-extension/src/common/constants.ts]: I've removed the code I added and just bumped `DevKitVersion` to `3.0.0`; it does the trick. Thank you for catching this!

File: devex/vscode-extension/.vscodeignore (hunk 2 above)
Comment (HernanGatta) [reply devex/vscode-extension/.vscodeignore]: Added.

File: devex/vscode-extension/package.json (hunk 3 above)
Comment (HernanGatta) [reply devex/vscode-extension/package.json]: Added.

File: devex/vscode-extension/src/common/constants.ts (hunk 1 above)
Comment (HernanGatta) [reply devex/vscode-extension/src/common/constants.ts]: This variable has been removed according to Brian's comment.

**Comment by HernanGatta** [comment]
bors try

**Comment by radhikaj** [comment]
please reopen when you can work on this again.

**Comment by dthaler** [comment]
@radhikaj Why close instead of keeping in Backlog?

**Pull request closed without merging**